### ISSUE TITLE
What is the effect of the choice of padding strategy on transformer performance?

### ISSUE THREAD

**Issue by kgreyy** [whitebox-research/excursions#6]
- DataLoader padding strategy
     - left vs right padding with attention masks
     - without attention masks
- Zero-padding

**Comment by zrkrlc** [comment]
By 'attention mask', you mean this, right? https://lukesalamone.github.io/posts/what-are-attention-masks/
